### ISSUE TITLE
Lots of unresolved links in Machines libraries

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#410]
**Reported by dietmarw on 14 Sep 2010 22:13 UTC**
I've checked today for unresolved links in the documentation and found loads of them in the various Machines libraries. This should be fixed in time for the release candidate at the end of September.

What I did was generating the html documentation of the MSL and then grepping for `modelica://` URIs. Those should normally be translated to `file://` but in case that a link does not work Dymola just leaves the original `modelica://` URI. 

You can find some "some" hints (not all of them might be valid bugs) where the documentation needs to be corrected in the attached output of grep.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/410

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 14 Sep 2010 22:27 UTC**
I forgot to mention that some of the links might not be working because the case-sensitivity. Since the MSL is and should be platform independent this needs to be fixed after all.

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 15 Sep 2010 21:44 UTC**
This issue is almost fixed by adding missing quotes around the href addresses, done in 2a864b2b675ab1bbd9917a4487495d76cf50836e.

However there are still some dead links left:
```
./Modelica_Electrical_Machines_Losses.html:67:Parameter record for <a href="modelica://Modelica.Electrical.Machines.Losses.DCMachines.Friction">Friction</a> losses.
./Modelica_Electrical_Machines_Losses.html:139:Parameter record for <a href="modelica://Modelica.Electrical.Machines.Losses.InductionMachines.AirGapRwithLosses">core losses of induction machines</a>
./Modelica_Electrical_Machines_Losses.html:140:and <a href="modelica://Modelica.Electrical.Machines.Losses.DCMachines.AirGapDCwithLosses">core losses of DC machines</a>.
./Modelica_Electrical_QuasiStationary_UsersGuide_Overview.html:45:      <a href="modelica://Modelica.Electrical.QuasiStationary.UsersGuide.Phasor">RMS phasors</a></li>
./Modelica_Electrical_QuasiStationary_UsersGuide_Overview.html:194:<a href="modelica://Modelica.Electrical.QuasiStationary.Examples.SeriesResonance">
./Modelica_Electrical_QuasiStationary_UsersGuide_Overview.html:305:<a href="modelica://Modelica.Electrical.QuasiStationary.Examples.SeriesResonance">
./Modelica_Electrical_QuasiStationary_MultiPhase_Interfaces.html:358:<a href="modelica://Modelica.Electrical.QuasiStationary.MutliPhase.Sources.VariableCurrentSource">VariableCurrentSource</a>,
```

**Comment by modelica-trac-importer** [comment]
**Comment by AHaumer on 16 Sep 2010 07:48 UTC**
resolved by 574c99aca7e9a379e43682c6cb6485d2ce253f53 - many thanks Dietmar!
